spec/support/capybara.rb

```diff
@@ -5,8 +5,27 @@
   Capybara.server_port = 9887 + ENV['TEST_ENV_NUMBER'].to_i
 end
 
+Capybara.register_driver :chrome_headless do |app|
```

Comment:
The default selenium chrome headless driver doesn't work in Docker. See https://stackoverflow.com/questions/50610316/capybara-headless-chrome-in-docker-returns-devtoolsactiveport-file-doesnt-exist